### ISSUE TITLE
Integrate database technology with backend

### ISSUE THREAD

**Issue by sarahdagger** [ubclaunchpad/floradora#21]
This issue depends on issue #3 (research what database technology to use). Once you have chosen a technology, we need to figure out how to integrate it with our node.js server. Create and implement four methods in the backend application, one for each of the CRUD (create, read, update, and delete) operations. These operations can just be on a simple test/dummy table for now - you don't need to implement the database schema yet. The purpose of this task is to demonstrate that we can interact with the database through the backend server.

**Comment by sarahdagger** [comment]
Please add dbdiagram.io generated Postgress code to `/backend/db` (you'll have to create the `db` folder)

**Comment by zeowang** [comment]
Had branch #32  merged into main.
